easybuild/easyconfigs/v/vcf-decision-tree/vcf-decision-tree-v0.0.2-Java-11-LTS.eb

```diff
@@ -0,0 +1,33 @@
+name = 'vcf-decision-tree'
+version = 'v0.0.2'
+versionsuffix = '-Java-%(javaver)s-LTS'
+
+homepage = 'https://github.com/molgenis/vip-decision-tree/'
+description = """Command-line application to classify variants in any VCF (Variant Call Format) file based on a decision tree."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+easyblock = 'Binary'
+
+#
+# Example URL:
+# https://github.com/molgenis/vip-decision-tree/
+#
+source_urls = [('http://github.com/molgenis/vip-decision-tree/releases/download/%s' % (version))]
+sources = [('vcf-decision-tree.jar' )]
```

Comment:
Dit is gevaarlijk: EasyBuild cacht z'n downloads en er zit geen versie nummer in de bestandsnaam. Bij een volgende v0.0.3 gaat EB hem dan niet opnieuw downloaden, want bestand is er al. Vervolgens zal de installatie crashen, omdat de checksum niet klopt.